src/fsharp/FSharp.Compiler.nuget/Microsoft.FSharp.Compiler.nuspec

```diff
@@ -15,70 +15,69 @@
             </group>
         </dependencies>
         <contentFiles>
-            <files include="any\any\default.win32manifest"                       buildAction="Content" copyToOutput="true" flatten="false"  />
-            <files include="any\any\Microsoft.FSharp.Targets"                    buildAction="Content" copyToOutput="true" flatten="false"  />
-            <files include="any\any\Microsoft.Portable.FSharp.targets"           buildAction="Content" copyToOutput="true" flatten="false"  />
-            <files include="any\any\Microsoft.FSharp.NetSdk.props"               buildAction="Content" copyToOutput="true" flatten="false"  />
-            <files include="any\any\Microsoft.FSharp.NetSdk.targets"             buildAction="Content" copyToOutput="true" flatten="false"  />
-            <files include="any\any\Microsoft.FSharp.Overrides.NetSdk.targets"   buildAction="Content" copyToOutput="true" flatten="false"  />
+            <files include="any\any\default.win32manifest"                       buildAction="Content" copyToOutput="true" flatten="false" />
+            <files include="any\any\Microsoft.FSharp.Targets"                    buildAction="Content" copyToOutput="true" flatten="false" />
+            <files include="any\any\Microsoft.Portable.FSharp.targets"           buildAction="Content" copyToOutput="true" flatten="false" />
+            <files include="any\any\Microsoft.FSharp.NetSdk.props"               buildAction="Content" copyToOutput="true" flatten="false" />
+            <files include="any\any\Microsoft.FSharp.NetSdk.targets"             buildAction="Content" copyToOutput="true" flatten="false" />
+            <files include="any\any\Microsoft.FSharp.Overrides.NetSdk.targets"   buildAction="Content" copyToOutput="true" flatten="false" />
         </contentFiles>
     </metadata>
     <files>
         <!--
             Why is this like this?
             This package is used to deploy the fsharp compiler tools as built product.
-            
+
             We need nuget package to merely deploy the contents of the lib directory.
             tools, build and runtime/native make unnecessary copies.
             this approach gives a very small deployment. Which is kind of necessary.
         -->
         <!-- assemblies -->
-        <file src="fsc.exe"                                     target="lib\netstandard1.6" />
-        <file src="fsi.exe"                                     target="lib\netstandard1.6" />
-        <file src="FSharp.Core.dll"                             target="lib\netstandard1.6" />
-        <file src="FSharp.Compiler.Private.dll"                 target="lib\netstandard1.6" />
-        <file src="FSharp.Build.dll"                            target="lib\netstandard1.6" />
-        <file src="FSharp.Compiler.Interactive.Settings.dll"    target="lib\netstandard1.6" />
+        <file src="bin\$configuration$\netcoreapp2.1\fsc.exe"                                     target="lib\netcoreapp2.1" />
```

Comment:
Is it not strange that this is a .NET Standard 2.0 NuGet package, but all the bins live under 'lib/netcoreapp2.1'?